diff --git a/libc/include/sys/system_properties.h b/libc/include/sys/system_properties.h
index d075859..291e775 100644
--- a/libc/include/sys/system_properties.h
+++ b/libc/include/sys/system_properties.h
@@ -38,7 +38,7 @@ __BEGIN_DECLS
 
 typedef struct prop_info prop_info;
 
-#define PROP_VALUE_MAX  92
+#define PROP_VALUE_MAX  126
 
 /*
  * Sets system property `key` to `value`, creating the system property if it doesn't already exist.
